e2e/cypress/integration/system_console/group_configuration_spec.js

```diff
@@ -0,0 +1,435 @@
+            addGroupSyncable('channel', (channelName) => {
+                // # Save the settings
+                savePage();
+
+                // # Reload the page
+                cy.visit(`/admin_console/user_management/groups/${groupID}`);
+
+                // # Test that the team persisted
+                teamOrChannelIsPresent(channelName);
+            });
+        });
+    });
+
+    describe('removing a team', () => {
+        it('does not remove a team without saving', () => {
+            cy.apiGetTeams().then((response) => {
+                // # Link a team
+                const team = response.body[0];
+                cy.apiLinkGroupTeam(groupID, team.id);
+
+                // # Reload the page
+                cy.visit(`/admin_console/user_management/groups/${groupID}`);
+
+                // * Check that the team was added to the view
+                teamOrChannelIsPresent(team.display_name);
+
+                // # Click remove and confirm
+                removeAndConfirm(team.display_name);
+
+                // # Reload the page
+                cy.visit(`/admin_console/user_management/groups/${groupID}`);
+
+                // * Check that the team is still visible
+                teamOrChannelIsPresent(team.display_name);
+            });
+        });
+
+        it('does remove a team when saved', () => {
+            cy.apiGetTeams().then((response) => {
+                // # Link a team
+                const team = response.body[0];
+                cy.apiLinkGroupTeam(groupID, team.id);
+
+                // # Reload the page
+                cy.visit(`/admin_console/user_management/groups/${groupID}`);
+
+                // * Check that the team was added to the view
+                teamOrChannelIsPresent(team.display_name);
+
+                // # Click remove and confirm
+                removeAndConfirm(team.display_name);
+
+                // # Save the settings
+                savePage();
+
+                // # Reload the page
+                cy.visit(`/admin_console/user_management/groups/${groupID}`);
+
+                // * Check that the team is no longer present
+                cy.get('.group-teams-and-channels-empty').should('be.visible');
+            });
+        });
+    });
+
+    describe('removing a channel', () => {
+        it('does not remove a channel without saving', () => {
+            cy.apiGetChannelByName('ad-1', 'aut-8').then((response) => {
+                // # Link a channel
+                const channel = response.body;
+                cy.apiLinkGroupChannel(groupID, channel.id);
+
+                // # Reload the page
+                cy.visit(`/admin_console/user_management/groups/${groupID}`);
+
+                // * Check that the channel was added to the view
+                teamOrChannelIsPresent(channel.display_name);
+
+                // # Click remove
+                cy.get(`button[data-testid='${channel.display_name}_groupsyncable_remove']`).click();
```

Comment:
```suggestion
                cy.findByTestId(`${channel.display_name}_groupsyncable_remove']`).click();
```

Reply:
Theres a few other places where you could this could be changed too, not a big deal though